add guidelines for custom containers (#2905)
nf-core/website: sites/docs/src/content/docs/guidelines/pipelines/recommendations/custom_containers.md

---
title: Custom containers
subtitle: What to do with custom containers that are hosted on docker.io or ghcr.io
menu:
  main:
    weight: 200
---

If a pipeline cannot be constructed with bioconda and therefore cannot be found on biocontainers, please contact `@core-team` on slack to discuss the best way to proceed.

> The current solution to ensure future reproducibitliy is mirror the container and host it on `quay.io` in the nf-core organisation.

Reviewer: ```suggestion
If possible, we prefer to mirror the container and host it on `quay.io` in the nf-core organisation to ensure future reproducibility (ie. that a container that the pipeline relies on will not be deleted by a 3rd party in the future).
```

> For that the following steps are necessary, and should be done by a core-team member.

Reviewer: ```suggestion
To do this, a core-team member should run the following commands:
```
